utils/doclint/generateDotnetApi.js

```diff
@@ -226,11 +226,6 @@ for (const [name, type] of modelTypes)
 for (const [name, literals] of enumTypes)
   renderEnum(name, literals);
 
-if (process.argv[3] !== '--skip-format') {
```

Comment:
Why was this removed?

Reply:
it's done here now: https://github.com/microsoft/playwright-dotnet/blob/main/build.sh#L69